### DISCUSSION TITLE
Install on macos

### DISCUSSION THREAD

**Discussion by Liquidrag** [Dineshkarthik/telegram_media_downloader#224] [Q&A] [answered]
can this be installed on a mac and if so, how?

**Comment by Dineshkarthik** [comment, marked as answer]
Yes it works on Mac. Follow the installation instructions from the Readme.

**Reply by ErfanKhodabakhsh** [reply]
please make a video as a tutorial for this code. it's very useful and no body knows there is a program for this goal!
